### DISCUSSION TITLE
[Unstable] Husks can stay permanently stunned if burned enough

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14672] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Husks don't naturally heal burns so you can keep them unconscious forever if they have enough burns

Not sure if this is intentional, a 0.1 burn regen might help

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Husks can now recover from burns naturally.
Closing discussion as resolved.
